apps/src/code-studio/pd/teachercon1819_registration/Confirmation.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import {
+  FormGroup,
+} from 'react-bootstrap';
+
+import Teachercon1819FormComponent from './Teachercon1819FormComponent';
+
+export default class Confirmation extends Teachercon1819FormComponent {
+  static associatedFields = [];
+
+  render() {
+    return (
+      <FormGroup>
+        {this.isTeacherApplication() &&
+          <p>
+            We're excited you're planning to join the Professional Learning
+            Program! You will receive more information about travel approximately
+            six weeks before TeacherCon. In the meantime, please <strong>do not</strong> book your
+            flight, and make sure to contact <a href="mailto:teacher@code.org">teacher@code.org</a> with any questions.
+            We look forward to meeting you this summer!
+          </p>
+        }
+
+        {this.isFacilitatorApplication() &&
+          <p>
+            We're excited you're planning to join us this summer! You will
+            receive more information about travel approximately six weeks before
+            TeacherCon. In the meantime, please <strong>do not</strong> book your flight, and make
+            sure to contact <a href="mailto:facilitators@code.org">facilitators@code.org</a> with any questions. We look
+            forward to meeting you this summer!
+          </p>
+        }
+
+        {this.isPartnerApplication() &&
+          <p>
+            We're excited you're planning to join us this summer! You will
+            receive more information about travel approximately six weeks before
+            TeacherCon. In the meantime, please <strong>do not</strong> book your flight, and make
```

Comment:
The first 2 sentences are the same in all cases. Might it make sense to split that out into its own paragraph, supplied once?

Reply:
Actually, one says "join the Professional Learning Program" and the other two say "join us this summer" :/

I'mma just leave it as is; if we get lots of changes coming in to the copy I'll probably try and dedup to make those easier.